pwa/src/templates/documentation/usageDocumentation/UsageDocumentationTemplate.module.css

```diff
@@ -11,6 +11,16 @@
   color: var(--web-app-color-text-grey) !important;
 }
 
+.codeblock {
+  --utrecht-code-block-background-color: var(--skeleton-color-grey-1);
+  --utrecht-code-block-padding-block-end: var(--skeleton-size-lg);
+  --utrecht-code-block-padding-block-start: var(--skeleton-size-lg);
+  --utrecht-code-block-padding-inline-end: var(--skeleton-size-lg);
+  --utrecht-code-block-padding-inline-start: var(--skeleton-size-lg);
```

Comment:
The `--skeleton` tokens are meant only as a starting point from which the themes are setup. They're never meant to be used within actual implementations.

We have the `--web-app` tokens, which you can use here.

Note: the `--web-app` tokens are being updated by the current theme, the `--skeleton` ones are not! So that means that these values are never updated when you switch from theme.